model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -939,7 +973,9 @@ class WorkspaceJsonSupport extends JsonSupport {
 
   implicit val WorkspaceShardStateFormat = rawlsEnumerationFormat(WorkspaceShardStates.withName)
 
-  implicit val WorkspaceDetailsFormat = jsonFormat18(WorkspaceDetails.applyOmitShardState)
+  implicit val WorkspaceStageFormat = rawlsEnumerationFormat(WorkspaceType.withName)
```

Comment:
`WorkspaceStageFormat` seems like an odd name for this.

Reply:
Ah, good eye, this was an artifact of a previous naming scheme, fixed.